packages/apps/job-launcher/server/src/modules/cron-job/cron-job.service.ts

```diff
@@ -120,7 +122,52 @@ export class CronJobService {
     await this.completeCronJob(cronJob);
   }
 
-  @Cron('*/3 * * * *')
+  @Cron('*/2 * * * *')
+  public async processJobModerationTasksCronJob() {
```

Comment:
I think this cron job is unnecessary. As far as I can see we have 4 cron jobs:
1. Create moderation task entity.
2. Send the data for moderation.
3. Parse results.
4. Send notification.

At least 1 and 2 could be combined.